contracts/InterchainTokenFactory.sol

```diff
@@ -123,6 +123,7 @@ contract InterchainTokenFactory is IInterchainTokenFactory, ITokenManagerType, M
      * @param initialSupply The amount of tokens to mint initially (can be zero), allocated to the msg.sender.
      * @param minter The address to receive the minter and operator role of the token, in addition to ITS. If it is set to `address(0)`,
      * the additional minter isn't set, and can't be added later. This allows creating tokens that are managed only by ITS, reducing trust assumptions.
+     * Reverts if the minter is interchainTokenService address (invalid).
```

Comment:
```suggestion
     * Reverts if the minter is the ITS address since it's already added as a minter.
```
consistent with how we refer elsewhere. explain why it reverts instead of just saying it's invalid

Reply:
got it. done